docs/spec/relayer/ChannelHandlers.tla

```diff
@@ -140,7 +140,64 @@ HandleChanOpenConfirm(chainID, chain, datagrams) ==
     \* otherwise, do not update the chain     
     ELSE chain 
     
+\* Handle "ChanCloseInit" datagrams
```

Comment:
The relayer algorithm for handling channel closing does not have an English specification in [ICS 018](https://github.com/cosmos/ics/tree/master/spec/ics-018-relayer-algorithms) (at least in the master branch). Any idea where is the English high-level spec for this algorithm? I wanted to cross-check it with your TLA+ code.

Reply:
You're right, it's missing in the ICS018, which is why channel closing wasn't added to the relayer spec until now. I've reverse engineered the datagram creation from the channel closure handlers.